vendors/nordic/boards/nrf52840-dk/ports/ble/iot_ble_hal_internals.h

```diff
@@ -140,6 +140,7 @@ static inline const BTStatus_t BTNRFError( const ret_code_t xErrCode )
             break;
 
         case NRF_ERROR_NO_MEM:
```

Comment:
Suggest adding a comment stating that this fall-though is intentional.

Reply:
Comment may seem redundant, alternative is to catch in its own case, but they both map to same BT API error status.